actions/contacts/models.go

```diff
@@ -1,22 +1,34 @@
 package contacts
 
 import (
+	"errors"
+
 	"github.com/bitcoin-sv/spv-wallet/engine"
-	"github.com/bitcoin-sv/spv-wallet/models"
 )
 
-// CreateContact is the model for creating a contact
-type CreateContact struct {
-	// Accepts a JSON object for embedding custom metadata, enabling arbitrary additional information to be associated with the resource
+// UpsertContact is the model for creating a contact
+type UpsertContact struct {
+	FullName string          `json:"fullName"`
 	Metadata engine.Metadata `json:"metadata" swaggertype:"object,string" example:"key:value,key2:value2"`
+
+	// optional
+	RequesterPaymail string `json:"requesterPaymail"`
```

Comment:
Maybe something more detailed?

```suggestion
	// optional paymail address owned by the user to bind the contact to. It is required in case if user has multiple paymail addresses
	RequesterPaymail string `json:"requesterPaymail"`
```